protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/non_copy_data_types/seq_inner.rs

```diff
@@ -227,3 +227,15 @@ impl<'a, T> From<Seq064K<'a, T>> for Vec<T> {
         v.0
     }
 }
+
+impl<'a, T> From<Vec<T>> for Seq0255<'a, T> {
```

Comment:
This is not safe as it can create invalid sequences, should be TryFrom and should check the input len or should stay under `#[cfg(feature = "prop_test")]` .

Reply:
I replaced the `From` impl with the `TryFrom` impl. Can you check out the impl and make sure it is correct? If it isn't, just lmk how I should change it.